src/components/collapsable/collapsable.tsx

```diff
@@ -0,0 +1,52 @@
+import { Component, Event, EventEmitter, h, Host, Prop } from "@stencil/core"
+
+@Component({
+  tag: "oui-collapsable",
+  styleUrl: "./collapsable.css",
+})
+export class Collapsable {
+  /**
+   * Set drawer drawerTitle
+   */
+  @Prop() public label: string
+
+  /**
+   * Set drawer position to right or left
+   */
+  @Prop() public expanded: boolean = false
+
+  /**
+   * Triggered when item collaspes
+   */
+  @Event() public collapse: EventEmitter
+
+  /**
+   * Triggered when the item expands
+   */
+  @Event() public expand: EventEmitter
+
+  public render() {
```

Comment:
can this be implemented using the existing tags for `<details>` and `<summary>` ? I feel like the entire collapsible thing is somewhat unnecessary outside of just adding some styling